src/Client.ts

```diff
@@ -1,20 +1,22 @@
 import ApiOptions from './Interfaces/ApiOptions';
 import HttpClient from './Interfaces/HttpClient';
 import resources from './Resources';
-import Auth from './Resources/Auth/auth';
-import Orders from './Resources/Orders/orders';
+import Search from './Resources/wacc/Search/search';
+import ReturnOrders from './Resources/wacc/Returns/returnOrders';
```

Comment:
Do we still need these here if they're WACC resources? Shouldn't that all be covered by the WACC object?